webplugin/js/app/kommunicate.js

```
@@ -421,16 +375,15 @@ $applozic.extend(true, Kommunicate, {
         }
         kmLocalStorage.removeItemFromLocalStorage('mckActiveConversationInfo');
         kmCookieStorage.deleteUserCookiesOnLogout();
-        window.Sentry && window.Sentry.close()
+        window.Sentry && window.Sentry.close();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)